fdbclient/ClientKnobs.cpp

```diff
@@ -314,6 +314,7 @@ void ClientKnobs::initialize(Randomize randomize) {
 	init( ENCRYPT_HEADER_AES_CTR_HMAC_SHA_AUTH_VERSION, 1 );
 
 	init( REST_KMS_ALLOW_NOT_SECURE_CONNECTION,     false ); if ( randomize && BUGGIFY ) REST_KMS_ALLOW_NOT_SECURE_CONNECTION = !REST_KMS_ALLOW_NOT_SECURE_CONNECTION;
+	init( SIM_KMS_VAULT_MAX_KEYS,                    4096 );
```

Comment:
Yeah I don't think you need to change it during a test, but varying this across test runs could be more useful than always having a static value

Reply:
Agreed. It might change the mapping of domainId <-> baseCipherId in some sense, I think the only additional work there is to preserve the parameter value across upgrade tests. If its OK, I can update it as a separate PR. 
Thanks